linkml_runtime/index/object_index.py

```diff
@@ -13,7 +13,7 @@
 from typing import Any, Union
 from collections.abc import Mapping, Iterator
 
-from linkml_runtime import SchemaView
+from linkml_runtime.utils.schemaview import SchemaView
```

Comment:
use full path for import (this was mostly to make it easier for me to find places where weird stuff was being imported from SchemaView)